portfolio/pom.xml

```diff
@@ -26,6 +26,12 @@
       <artifactId>javax.servlet-api</artifactId>
       <version>4.0.1</version>
     </dependency>
+    
+<dependency>
```

Comment:
[nit] indentation, this should align with the <dependency> blocks above and below